### DISCUSSION TITLE
aptio bios utility 2020 version 2.19.1268 where lan / network / ipxe boot change?

### DISCUSSION THREAD

**Discussion by doguibnu** [ipxe/ipxe#263] [General]
Hello!
How are you?

I have a mini PC with aptio utility bios 2020 version 2.19.1268 and I need to change boot mode for network / Lan / ipxe. I navigate inside all options and I cannot see or change for network boot mode. Please, someone knows how can I fix this?
There are some pictures from this Bios:

Thank you attention and help

Douglas

![01-main1-02-15 at 14 08 07](https://user-images.githubusercontent.com/38897311/108074970-93873500-7048-11eb-9737-7bf3ce2a45c4.jpeg)
![02-advanced-2021-02-15 at 14 08 08](https://user-images.githubusercontent.com/38897311/108074994-9bdf7000-7048-11eb-980d-55d72dedee8e.jpeg)
![03-chipset2021-02-15 at 14 08 08](https://user-images.githubusercontent.com/38897311/108075016-a13cba80-7048-11eb-93b7-6d1ed6c996de.jpeg)
![04-bios2021-02-15 at 14 08 09](https://user-images.githubusercontent.com/38897311/108075031-a6016e80-7048-11eb-9232-7b7bce8e1ed8.jpeg)
![05-security2021-02-15 at 14 12 39](https://user-images.githubusercontent.com/38897311/108075061-ae59a980-7048-11eb-80e1-c40570b9e01e.jpeg)
![14-fast-boot2021-02-15 at 15 46 50](https://user-images.githubusercontent.com/38897311/108075102-b9143e80-7048-11eb-9d63-8b08d08674c8.jpeg)
![15-noth-bridge2021-02-15 at 15 45 15](https://user-images.githubusercontent.com/38897311/108075115-bd405c00-7048-11eb-9ca3-9788eba909b9.jpeg)
![16-north-bridge-openImage 2021-02-15 at 15 45 26](https://user-images.githubusercontent.com/38897311/108075121-c0d3e300-7048-11eb-8ab2-ee4a127c7a58.jpeg)
![17-soth-bridgemage 2021-02-15 at 15 45 35](https://user-images.githubusercontent.com/38897311/108075133-c3ced380-7048-11eb-9339-2f71df4e449f.jpeg)

**Comment by NiKiZe** [comment]
Is iPXE included in the firmware or flashed onto any PCI device?

The answer is probably no in both cases. Probably due to confusion about iPXE and PXE or EFI network stack.
You should check with the hardware manufacturer.

**Comment by habetsm-xilinx** [comment]
Go to "Boot Option #1" and press return.
It will show you all possibilities. If the BIOS has detected a network card that can PXE boot it will show up.

If no network card shows up here you may be able to enable that for your hardware. For example, if it is a PCI card go to "PCI Subsystem Setting" and ensure that BootROM support is enabled for the slot.
After enabling such a PCI option you have to power cycle the PC, boot it all the way into grub, and then reboot it again before the BIOS will have detected the device. Check "Boot Option #1" again.

While in the Boot menu disable "Quiet Boot", in a good setup the network card should print it's ROM header.

**Reply by doguibnu** [reply]
Hello @habetsm-xilinx 

Thanks for answer friend!
I will check these options
When done or not I tell here.

Thank you

Douglas

**Reply by doguibnu** [reply]
@habetsm-xilinx 

In this Option: PCI Subsystem Settings has not BootRom as show in picture link 

https://imgur.com/2KkIVRS

I do not believe that bios does not have lan boot option!

thanks

**Reply by NiKiZe** [reply]
Again, contact your manufacturer of the hardware. I have come across boards that does not have a ROM in EFI only mode, cheap Chinese stuff. There is no guarantee that it is there.

And again note that the use of iPXE is rare in firmware. PXE ROM is in most cases not iPXE. But even if it was iPXE can not help you with the firmware, only the board manufacturer can do that (they are in this case using Aptio BIOS which they have modified)

**Reply by doguibnu** [reply]
Hello @NiKiZe 

How are you?
its impossible contact the manufacturer. Because you are right about cheap Chinese stuff. We will pass the device to other person and try found other with more attention and more quality. Ask to seller about the boot options. I really think was common lan/pxe/rom boot option on all mobos. At least desktop Mobo there is this option for years. 

thank you attention and help

Douglas
